style.css

```diff
@@ -0,0 +1,313 @@
+    width: 24px;
+    height:24px;
+    display: inline-block;
+}
+.grid-item:nth-child(6n):not(:nth-child(6)){
+    color: blue;
+}
+.grid-item:nth-child(6n):not(:nth-child(6)):after{
+    content: "";
+    background-image: url("data:image/svg+xml,%3Csvg width='14' height='14' viewBox='0 0 14 14' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M7.86514 0.0289874C7.72259 0.0704427 7.53305 0.251339 7.47619 0.400231C7.35778 0.710278 7.47341 1.0497 7.75352 1.21435L7.89625 1.29825L9.81958 1.3132L11.7429 1.32816L8.98501 4.07988C7.04669 6.01384 6.21088 6.86717 6.1725 6.95125C6.06749 7.18143 6.11675 7.48045 6.29017 7.66538C6.47232 7.85961 6.7879 7.91671 7.04381 7.80171C7.1482 7.75478 7.79384 7.12613 9.92179 4.99938L12.6652 2.25752L12.6735 4.17069L12.6819 6.08386L12.7658 6.22659C12.9304 6.50669 13.2698 6.62233 13.5799 6.50391C13.7356 6.44442 13.9098 6.25623 13.953 6.10075C13.9715 6.03391 13.9826 4.97294 13.9825 3.25736L13.9823 0.520589L13.9121 0.371039C13.8295 0.195168 13.7745 0.141359 13.5936 0.0592862C13.4618 -0.000503923 13.4128 -0.00149096 10.7078 0.000513012C9.19435 0.00164959 7.91515 0.0144511 7.86514 0.0289874ZM2.06379 2.20159C1.86812 2.22947 1.60446 2.31047 1.3956 2.4069C0.776166 2.69278 0.329339 3.19188 0.0987331 3.85556L0 4.1397V8.08783V12.036L0.0987331 12.3201C0.227765 12.6914 0.391941 12.9639 0.650872 13.2365C1.01183 13.6165 1.44678 13.856 1.95118 13.9525C2.28363 14.0161 9.50281 14.0157 9.83541 13.9521C10.7338 13.7802 11.4554 13.1119 11.717 12.2094L11.7846 11.9761V8.11774C11.7846 5.99562 11.7736 4.22001 11.7603 4.17195C11.7364 4.08616 11.6972 4.12325 9.60675 6.20834C7.14626 8.66249 7.33965 8.50564 6.77462 8.50531C6.51856 8.50516 6.42934 8.49263 6.29149 8.43751C5.96215 8.30572 5.67447 8.01877 5.54374 7.69158C5.48745 7.55076 5.47485 7.46223 5.47426 7.2039C5.47357 6.89908 5.47737 6.88051 5.5851 6.66256C5.69188 6.44652 5.78643 6.3473 7.79106 4.34767C9.25668 2.88567 9.87597 2.24906 9.85395 2.22702C9.82326 2.19633 2.27358 2.17168 2.06379 2.20159Z' fill='%237C5BFF'/%3E%3C/svg%3E%0A");
+    width: 14px;
+    height:14px;
+    display: inline-block;
+}
+.navbar-nav {
+    width: 3.875rem;
+    height: 8.313rem;
+    display: flex;
+    flex-direction: column;
+    margin: 10.813rem 0 0 1.813rem;
+}
+.navbar-nav_bottom {
+    width: 3.875rem;
+    height: 8.313rem;
+    display: flex;
+    flex-direction: column;
+    margin: 0 0 0 1.813rem;
+}
+.navbar-nav li:nth-child(2),
+.navbar-nav li:nth-child(3){
+    margin-top: 8px;
+}
+
+.navbar-nav > li{
+    text-decoration: none;
+    list-style: none;
+}
+.navbar-nav_bottom > li{
+    text-decoration: none;
+    list-style: none;
+}
+.navbar-nav .nav-item {
+    border-radius: 0.5rem;
+    width: 64px;
+    height: 39px;
+}
+.nav-item{
```

Comment:
nav items style adjust
add a `.selected` class name for selected
and :hover to everyone

Reply:
selected item should not be hoverable